sn06.yml

```diff
@@ -37,17 +37,6 @@
       - "galaxyproject.eu"
       - "*.galaxyproject.eu"
       - "*.ep.interactivetool.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.live.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.test.usegalaxy.eu"
-      - "*.aqua.usegalaxy.eu"
-      - "*.interactivetoolentrypoint.interactivetool.aqua.usegalaxy.eu"
-      - "*.ecology.usegalaxy.eu"
```

Comment:
Here are the subsomains

Reply:
Are they still used, I think not because they are only a few and it should all be covered by the wildcard cert *.usegalaxy.eu

Reply:
I was wondering why there are so few, but maybe we only support these few subdomains for ITs?

Reply:
Yes, we only supported a few.